File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/UbicacionLoteRequest.java
package com.farmacia.sanidadbackend.dto;

import jakarta.validation.constraints.Min;
import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class UbicacionLoteRequest {
    @NotNull
    private Long loteDetalleId;
    @NotNull
    private Long rackId;
    @NotNull @Min(0)
    private Integer nivel;
    @NotNull @Min(0)
    private Integer columna;
    @NotNull @Min(0)
    private Integer profundidadIndex;
    @NotNull @Min(1)
    private Integer cantidad;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/UbicacionLoteRequest.kt
package com.sanidad.movil.data.remote.dto

data class UbicacionLoteRequest(
    val loteDetalleId: Long,
    val rackId: Long,
    val nivel: Int,
    val columna: Int,
    val profundidadIndex: Int,
    val cantidad: Int
)